### PR TITLE
Fixes!

### PR DESCRIPTION
**Pull request by sgdc3**

### CODE REVIEW

**Comment by Xephi** [comment]
Some test done ? does it work ?

**Comment by sgdc3** [comment]
It seems to work.

File: src/main/java/fr/xephi/authme/listener/AuthMePlayerListener.java [hunk 1]
```diff
@@ -398,6 +401,64 @@ public void run() {
...
+    public void onPlayerJoin(PlayerJoinEvent e) {
+        final PlayerJoinEvent event = e;
+
+        if (event.getPlayer() == null) {
+            return;
+        }
+
+        // Shedule login task so works after the prelogin
+        // (Fix found by Koolaid5000)
+        Bukkit.getScheduler().runTask(plugin, new Runnable(){
```
Comment (Xephi) [inline src/main/java/fr/xephi/authme/listener/AuthMePlayerListener.java]: Why don't use scheduleDelayedTask ?

File: src/main/java/fr/xephi/authme/listener/AuthMePlayerListener.java [hunk 2]
```diff
@@ -737,6 +760,35 @@ public void onPlayerInventoryClick(InventoryClickEvent event) {
     }
 
     @EventHandler(ignoreCancelled = true, priority = EventPriority.LOWEST)
+    public void playerHitPlayerEvent(EntityDamageByEntityEvent event) {
```
Comment (Xephi) [inline src/main/java/fr/xephi/authme/listener/AuthMePlayerListener.java]: Duplicate :O

**Pull request merged**